web/src/components/RawDataEditor/locales/zh-CN.ts

```diff
@@ -16,5 +16,5 @@
  */
 export default {
   'component.rawDataEditor.tip': '目前暂不支持编辑',
-  'component.rawDataEditor.title': '元数据编辑器',
+  'component.rawDataEditor.title': 'CodeMirror',
```

Comment:
English in  zh-CN.ts🤔

Reply:
I guess there is no suitable word for that in cn

Reply:
Yes.

BTW, I want to make sure most of users know what CodeMirror is 😂

Does this word make sense to you? :)

Reply:
Maybe I need add some descriptions for CodeMirror and  still keeps using this word? 😊

Reply:
That would be good for everyone then, who are having problem with this word right now.

Reply:
> Yes.
> 
> BTW, I want to make sure most of users know what CodeMirror is 😂
> 
> Does this word make sense to you? :)

yes :)